src/hera.cpp

```diff
@@ -73,6 +74,40 @@ bool hasWasmPreamble(vector<uint8_t> const& _input) {
     _input[7] == 0;
 }
 
+bool deploySystemContract(string & code, evmc_address addr) {
+  evmc_message create_message;
+  create_message.input_data = reinterpret_cast<const uint8_t *>(code.c_str());
+  create_message.destination = addr;
+
+  //Prepare message call to deploy system contract
```

Comment:
actually it might make sense to just create a system contract map (addresses to function pointers)